fastai/callback/core.py

```diff
@@ -165,15 +166,15 @@ def __init__(self,
         ds_idx:int=1, # Index of dataset, 0 for train, 1 for valid, used if `dl` is not present
         dl:DataLoader=None, # `DataLoader` used for fetching `Learner` predictions
         with_input:bool=False, # Whether to return inputs in `GatherPredsCallback`
-        with_decoded:bool=False, # Whether to return predicted classes
-        cbs:list=None, # `Callback` list to add to the `Learner`
+        with_decoded:bool=False, # Whether to return decoded predictions
+        cbs:(Callback,list)=None, # `Callback` to temporarily remove from `Learner`
```

Comment:
```suggestion
        cbs:(Callback,list)=None, # `Callback`(s) to temporarily remove from `Learner`
```

Reply:
I think the (s) after the hotlink looks a bit silly, which is why I left it off.